package.json

```diff
@@ -48,7 +46,6 @@
     "dev:server": "yarn run kill-server && lerna run --stream --parallel dev:builder --concurrency 1 --scope @budibase/backend-core --scope @budibase/worker --scope @budibase/server",
     "dev:built": "cd packages/server && yarn dev:stack:up && cd ../../ && lerna run --stream --parallel dev:built",
     "test": "lerna run --stream test --stream",
-    "test:pro": "bash scripts/pro/test.sh",
```

Comment:
Out of curiosity what happens when we run test, build etc with an empty pro dir? 
Does lerna just ignore the package in that case?

Reply:
It was kind of nice that we were able to have a dedicated job in the pipeline for the pro tests (especially as they are somewhat flaky at present) - might be able to replace that with `--scope` definitions instead

Reply:
Do you think it's worth having pro as a separate task? As you say, we can use `--scope` and `--exclude` for it.
On the first question, yarn workspace will be checking for valid packages in `package/*`, so if it does not exists, nothing happen. If this is the case, it will fetch `@budibase/pro` from the repo.
At least, this is what I hope it will happen :D Still testing in progress

Reply:
Added back